certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/text/TextUtil.java

```diff
@@ -17,15 +17,17 @@
 @Component
 public class TextUtil {
 
-  private static final String[] PROBLEM_HYPHENS = {
-      "\u2010", // hyphen
-      "\u2011", // non-breaking hyphen
-      "\u2012", // figure dash
-      "\u2013", // en dash
-      "\u2014", // em dash
-      "\u2015", // horizontal bar
-      "\u2212"  // minus
-  };
+  private static final Map<String, String> PROBLEM_CHARACTERS_MAPPING = Map.of(
+      "\u2010", "-", // hyphen
+      "\u2011", "-", // non-breaking hyphen
+      "\u2012", "-", // figure dash
+      "\u2013", "-", // en dash
+      "\u2014", "-", // em dash
+      "\u2015", "-",  // horizontal bar
+      "\u2192", "->",  // right arrow
+      "\u2190", "<-",   // left arrow
+      "\u2194", "<->"   // left right arrow
```

Comment:
There are tests for this in PdfField, please add a test for the arrows before merging.